### PR TITLE
(#2186177) pstore: fix crash and forward dummy arguments instead of NULL

### PR DESCRIPTION
**Pull request by msekletar**
[msekleta: in our version of systemd "const char path*" argument of path_join() can't be NULL. Here we don't really want any subdirs paths passed into move_file(), but we can't just pass NULL pointers because they will be forwarded to path_join(). Hence, let's just pass "/" instead.]

RHEL-only

Related: #2186177

### CODE REVIEW

File: src/pstore/pstore.c [hunk 1]
```diff
@@ -366,7 +366,7 @@ static int run(int argc, char *argv[]) {
 
         /* Move left over files out of pstore */
         for (size_t n = 0; n < list.n_entries; n++)
-                (void) move_file(&list.entries[n], NULL, NULL);
+                (void) move_file(&list.entries[n], "/", "/");
```
Comment (dtardon) [inline src/pstore/pstore.c:369]: Nice catch! The 3rd arg. could remain `NULL`, but it doesn't matter...

**Review by dtardon — DISMISSED** [review]
LGTM

**Comment by dtardon** [comment]
We need this for 8.7, 8.8 and 8.9 too. Are you going to handle it?

**Comment by jamacku** [comment]
@dtardon I think for 8.7 is too late. 8.8 and 8.9 should be enough.

**Review by jamacku — COMMENTED** [review]
@msekletar, @dtardon Could we also include patches for the new commit linter:

- https://github.com/redhat-plumbers/systemd-rhel8/commit/a81ace51e1efa2313fcad18f2ef5c6c6cd167240
- https://github.com/redhat-plumbers/systemd-rhel8/commit/7e0eb456a99229d19b8e42491c1ddb7b8050903f

**Comment by msekletar** [comment]
@jamacku I will submit PRs for other branches as well.

**Review by jamacku — APPROVED** [review]

**Comment by msekletar** [comment]
> @jamacku I will submit PRs for other branches as well.

@jamacku I think we have PRs for all necessary branches now. PTAL.

**Pull request merged**